build.gradle

```diff
@@ -22,7 +26,27 @@ dependencies {
     testCompile group: 'org.springframework', name: 'spring-test', version: '5.1.4.RELEASE'
     testCompile group: 'org.testng', name: 'testng', version: '7.0.0-beta1'
     compile group: 'com.opencsv', name: 'opencsv', version: '4.1'
+    testCompile "io.cucumber:cucumber-java:${cucumberVersion}"
+    testCompile "io.cucumber:cucumber-java8:${cucumberVersion}"
+    testCompile "io.cucumber:cucumber-testng:${cucumberVersion}"
+    testCompile "io.cucumber:cucumber-spring:${cucumberVersion}"
+}
 
+configurations {
+    cucumberRuntime {
+        extendsFrom testRuntime
+    }
+}
+
+task cucumber() {
```

Comment:
can you investigate also ability to pass custom cucumber.options via system properties ?